internal/server/polling/main_test.go

```diff
@@ -0,0 +1,76 @@
+package polling
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"sync/atomic"
+	"testing"
+
+	. "github.com/onsi/gomega"
```

Comment:
I'm not a fan of dot imports either. I quite like the gomega API, so I live with them for the sake of keeping those expressions easier to read -- `g.Expect(operation).To(Succeed())` rather than `g.Expect(operation).To(gomega.Succeed())`.

In 290a9c5d2772dfeecf7c55dccda8e82be9c581e4 I've changed the dot import in two files, then used helpers for common cases in one of them. Have a look and see what you think. I am equally happy with either, myself.

Reply:
I like it much more. :heart: